### PR TITLE
Cockpit 303

### PR DESCRIPTION
**Pull request by martinpitt**
Preview: https://martinpitt.github.io/cockpit-project.github.io/blog/cockpit-303.html

 - [x] bodhi
 - [ ] ~release c-podman~ no release notes, ignore in this round

### CODE REVIEW

File: _posts/2023-10-18-cockpit-303.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+![screenshot of detect missing appstream metadata]({{ site.baseurl }}/images/303-apps-detect-missing-appstream-metadata.png)
+
+![screenshot of detect missing appstream metadata]({{ site.baseurl }}/images/303-apps-detect-missing-appstream-metadata-2.png)
+
+_Thanks to [leomoty](https://github.com/leomoty/) for designing this improvement!_
+
+## Machines: Add SSH keys to VM creation dialog
+
+The provided keys are stored in the `~/.ssh/authorized_keys` file of the designated non-root user, enabling immediate SSH access to the user account right after creating the VM.
```
Comment (allisonkarlitskaya) [inline _posts/2023-10-18-cockpit-303.md:29]: Could use a bit more introductory verbiage before jumping right in here.  Like:

"When creating a new VM, cockpit-machines now offers the option to provide your SSH public key."

File: _posts/2023-10-18-cockpit-303.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 303 and cockpit-machines 300:
+
+
+## Apps: Detect missing AppStream metadata
+
+If the [AppStream](https://www.freedesktop.org/software/appstream/docs/index.html) metadata is not installed, the "Apps" page cannot show available Cockpit extensions. The page detects this now, and offers the user to install the metadata.
```
Comment (allisonkarlitskaya) [inline _posts/2023-10-18-cockpit-303.md:19]: s/the user//

**Review by allisonkarlitskaya — APPROVED** [review]
I'd be happy enough with this as it is (and I double-checked the links).
A few suggestions below.

You mentioned as a point in the release tag that we now warn on connecting to other hosts via SSH.  Do you want to draw attention to this in the release notes as well?

Also: looks like we could use an update of the flatpak.  It didn't get released for 302, either.

File: _posts/2023-10-18-cockpit-303.md (hunk 1 above)
Comment (martinpitt) [reply _posts/2023-10-18-cockpit-303.md]: Good idea, thanks! Added that and adjusted the next sentence accordingly.

File: _posts/2023-10-18-cockpit-303.md (hunk 2 above)
Comment (martinpitt) [reply _posts/2023-10-18-cockpit-303.md]: Done, thanks!

**Comment by martinpitt** [comment]
> You mentioned as a point in the release tag that we now warn on connecting to other hosts via SSH. Do you want to draw attention to this in the release notes as well?

I wanted to write something into the tag, then found the apps improvement, and then felt like leaving it in. But it's not really a new feature.

> Also: looks like we could use an update of the flatpak. It didn't get released for 302, either.

https://github.com/flathub/org.cockpit_project.CockpitClient/pull/37

File: _posts/2023-10-18-cockpit-303.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+![screenshot of detect missing appstream metadata]({{ site.baseurl }}/images/303-apps-detect-missing-appstream-metadata.png)
+
+![screenshot of detect missing appstream metadata]({{ site.baseurl }}/images/303-apps-detect-missing-appstream-metadata-2.png)
+
+_Thanks to [leomoty](https://github.com/leomoty/) for designing this improvement!_
+
+## Machines: Add SSH keys to VM creation dialog
+
+When creating a new VM, cockpit-machines now offers the option to provide your SSH public key. These are stored in the `~/.ssh/authorized_keys` file of the designated non-root user, enabling immediate SSH access to the user account right after creating the VM.
```
Comment (allisonkarlitskaya) [inline _posts/2023-10-18-cockpit-303.md:29]: 'your SSH public key' = singular
'these' = plural?

**Review by allisonkarlitskaya — CHANGES_REQUESTED** [review]
One very minor nag left.

File: _posts/2023-10-18-cockpit-303.md (hunk 3 above)
Comment (martinpitt) [reply _posts/2023-10-18-cockpit-303.md]: Fixed, thanks!

**Review by allisonkarlitskaya — APPROVED** [review]
Thanks!

**Pull request merged**